packages/hasura-storage-js/src/hasura-storage-client.ts

```diff
@@ -33,29 +35,58 @@ export class HasuraStorageClient {
   }
 
   /**
-   * Use `nhost.storage.upload` to upload a file. The `file` must be of type [`File`](https://developer.mozilla.org/en-US/docs/Web/API/File).
+   * Use `nhost.storage.upload` to upload a file. 
+   * 
+   * It's possible to use [`File`](https://developer.mozilla.org/en-US/docs/Web/API/File) or [`FormData`](https://developer.mozilla.org/en-US/docs/Web/API/FormData) to upload a file. The `File` instance is only available in the browser while `FormData` with [`form-data`](https://www.npmjs.com/package/form-data) works both in the browser and in NodeJS (server).
    * 
-   * If no `bucket` is specified the `default` bucket will be used.
+   * If no `bucketId` is specified the bucket `default` is used.
    *
    * @example
+   * 
+   * Example to upload a file from the browser using `File`.
+   * 
    * ```ts
    * await nhost.storage.upload({ file })
    * ```
+   * 
+   * Example to upload a file from the browser using `File` to a specific Bucket.
    * 
     @example
    * ```ts
    * await nhost.storage.upload({ file, bucketId: '<Bucket-ID>' })
    * ```
+   * 
+   * Example to upload a file using `FormData` from the server.
    *
+   * @example
+   * ```ts
+   * import FormData from 'form-data'
+   * 
+   * const fd = new FormData() 
+   * fd.append('file', fs.createReadStream('./tests/assets/sample.pdf'))
+   * 
+   * await storage.upload({
+   *   formData: fd
+   * })
+   * ```
+   * 
    * @docs https://docs.nhost.io/reference/javascript/storage/upload
    */
   async upload(params: StorageUploadParams): Promise<StorageUploadResponse> {
-    const file = new FormData()
-    file.append('file', params.file)
+    let body
```

Comment:
this body is always a type of `FormData` isn't it?

Reply:
Yes.